test/framework/filetools.py

```diff
@@ -2888,6 +2888,24 @@ def test_locate_files(self):
         error_pattern = r"One or more files not found: 2\.txt \(search paths: \)"
         self.assertErrorRegex(EasyBuildError, error_pattern, ft.locate_files, ['2.txt'], [])
 
+    def test_create_unused_dir(self):
```

Comment:
missing docstring

Reply:
Hm, I don't think any of the docstrings here is useful, but added for consistency